src/pykeen/losses.py

```diff
@@ -384,25 +546,97 @@ def __init__(self, margin: float = 9.0, adversarial_temperature: float = 1.0, re
 
         .. note:: The default hyperparameters are based the experiments for FB15K-237 in [sun2019]_.
         """
+        # TODO: The "temperature" parameter here is more an reciprocal temperature, following the standard nomenclature
+        #  for softmax temperature
```

Comment:
@cthoyt I remember we discussed this at some point already.

Reply:
I think the end of the last discussion was we don't want to change it and worry about comparability of data yet